### DISCUSSION TITLE
Sale start date and End date Not working.

### DISCUSSION THREAD

**Discussion by Readbetweenthellime** [woocommerce/woocommerce#36191] [WooCommerce HPOS]
Hi! I'm trying to upload the sale price via CSV file. However, the sale price date is not reflected on the product page and I think it is not working.  Can someone help me? Thank y
[Test.csv](https://github.com/woocommerce/woocommerce/files/10305666/Test.csv)
ou.

**Comment by jorgeatorres** [comment]
Hi @Readbetweenthellime! Is this still a problem? If so, could you please [open a GH issue](https://github.com/woocommerce/woocommerce/issues/new/choose) in our repo and we'll take it from there?
